lib/chains/viacoin.py

```diff
@@ -0,0 +1,213 @@
+'''Chain-specific Viacoin code'''
+
+############################################
+# Code here is taken from Vialectrum       #
+# https://github.com/vialectrum/vialectrum #
+############################################
+
+from cryptocur import CryptoCur, hash_encode, hash_decode, rev_hex, int_to_hex, sha256, Hash
+import os
+try:
+    from ltc_scrypt import getPoWHash
+except ImportError:
+    from scrypt import scrypt_1024_1_1_80 as getPoWHash
+
+
+class Viacoin(CryptoCur):
+    chain_index = 14
+    coin_name='Viacoin'
+    code = 'VIA'
+    p2pkh_version = 71
+    p2sh_version = 33
+    wif_version = 199
+    ext_pub_version = '0488b21e'
+    ext_priv_version = '0488ade4'
+
+    DUST_THRESHOLD = 0
+    MIN_RELAY_TX_FEE = 100000
+    RECOMMENDED_FEE = MIN_RELAY_TX_FEE
+    COINBASE_MATURITY = 100
+
+    block_explorers = {
+        'explorer.viacoin.org': 'http://explorer.viacoin.org/tx/',
+        'viacoinexplorer.com': 'http://viacoinexplorer.com/tx/'
+    }
+
+    base_units = {
+        'VIA': 8,
+        'mVIA': 5,
+        'bits': 2
+    }
+
+    chunk_size = 2016
+
+    DEFAULT_PORTS = {'t':'50001', 's':'50002', 'h':'8081', 'g':'8082'}
+
+    DEFAULT_SERVERS = {
+        'server.vialectrum.org': DEFAULT_PORTS,
+        'vialectrum.cryptoservices.net': DEFAULT_PORTS,
```

Comment:
Nuke this one, I cant contact the developer and I dont think he upgraded to 0.10.6 as mandated.